server/models/notification.js

```diff
@@ -0,0 +1,29 @@
+const mongoose = require("mongoose");
+
+const notificationSchema = new mongoose.Schema({
+  recipient: {
+    type: mongoose.Schema.Types.ObjectId,
+    ref: "User",
+  },
+  thread: {
+    // type: mongoose.Schema.Types.ObjectId,
+    // ref: "Thread",
```

Comment:
Should we a ref to a request object probably?

Reply:
I'm just not sure where we save thread with code review request